lib/client/ClientRenderingWorkflow.js

```diff
@@ -72,10 +71,14 @@ var ClientRenderingWorkflow = {
                 return originalHandler.apply(router, allParams);
             });
 
-            var putLayoutBackToNormalAndExecuteLayoutInstructions = Promise.method(function(view) {
+            var prepareLayoutForRoute = Promise.method(function(view) {
                 layout.backToNormal();
                 layoutDelegate.replayInstructions();
 
+                if (isFirstPageLoad) {
+                    layout.enterDOM();
```

Comment:
Sure. v0.26.0 changed the order in which rendering occurred. this change fixes the rendering order. check out ClientRenderingOrderSpec. it documents what the order should be on the first request and all other requests